pkgs/applications/version-management/yadm/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchurl, git, bash }:
```

Comment:
bash is part of stdenv, so it's always available. If you want to specify dependencies that are available in the build environment, you have to specify them in the `buildInputs` argument to `stdenv.mkDerivation`.

The first line of the file just brings those packages into the scope of this package (it defines an anonymous function that takes an attribute set with the specified attributes that is called by `callPackage` from `pkgs/top-level/all-packages.nix`). Adding packages here but not somewhere below in the call to `stdenv.mkDerivation` doesn't change anything in the package building process.